src/test/java/com/google/spanner/liquibase/LiquibaseTests.java

```diff
@@ -97,13 +107,13 @@ public TestHarness.Connection getSpannerEmulator(Dialect dialect) throws SQLExce
     switch (dialect) {
```

Comment:
Do you need this switch statement? Why cannot the dialect be passed directly in to the `TestHarness.useSpannerEmulator(Dialect)` method?